lib/Plan.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+interface Plan
```

Comment:
change methods to getSmth() 
ex. addServersApiConst feels like an action

Reply:
Done